installing/uninstall-acs.adoc

```diff
@@ -0,0 +1,21 @@
+[id="uninstall-acs"]
+= Uninstalling {product-title}
+include::modules/common-attributes.adoc[]
+:context: uninstall-acs
+
+toc::[]
+
+[role="_abstract"]
+When you install {product-title}, it creates:
+
+* A namespace called `stackrox` and installs {product-title} and its components in that namespace
+* `PodSecurityPolicy` and Kubernetes Role-Based Access Control (RBAC) objects for all components
+* Additional labels on namespaces, for use in generated network policies
+//TODO: Add link to generate network policies
+* An Application CRD (if it does not exist)
+
+include::modules/delete-acs-namespace.adoc[leveloffset=+1]
```

Comment:
So this module to delete the stackrox namespace comes first, but the next module uses commands that pass in -n stackrox. Should this module be last instead?

Reply:
Yeah! good spotting. I'll check with engineering.